css/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 1.5;
+}
+
+header {
+  font-size: 20px;
+  margin: 0 auto;
+  text-align: center;
+  background-color: lightgreen;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
```

Comment:
Really nice 👍🏾  work here on making the image fully responsive.
This is perfect and it works really well.